Contributions-exercise.md

```diff
@@ -0,0 +1,3 @@
+# Contribution Exercise
+
+Make changes to this file as part of the contribution exercise. Feel free to add markdown content with correct or incorrect syntax and observe the results of the markdown linter. Do you find the wrod with the wrong spelling and can you fix it?
```

Comment:
If several persons are taking the coarse at the same time, How will this work? If the first person has already made the changes (for instance found wrod, changes it word) and the PR request is approved, what would the next person practice on? May be we should encourage to just feed the next incorrect syntax, if it is not there?

Reply:
This typo is not meant to be corrected. The contribution guide states that such PRs will not get merged, thus the typo remains for everybody to propose a fix for: https://github.com/ExpertLearningLab/foss-learning/blob/main/CONTRIBUTING.md?plain=1#L32

Reply:
Great, thanks for the clarification